includes/urls.php

```diff
@@ -49,6 +49,26 @@ public static function event_create(): string {
 		return gp_url( '/events/new/' );
 	}
 
+	/**
+	 * Returns the absolute URL to the image for the event.
+	 *
+	 * @param int $event_id The event ID.
+	 *
+	 * @return string
+	 */
+	public static function event_image( int $event_id ): string {
+		return trailingslashit( gp_url_public_root() ) . "events/image/$event_id";
```

Comment:
I think we were using this convention:
```suggestion
		return home_url( gp_url( "events/image/$event_id" ) );
```

Reply:
I only see one occurrence, but it is ok for me to use it!